analytics/src/main/java/com/segment/analytics/integrations/AttemptGoalPayload.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014 Segment, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.segment.analytics.integrations;
+
+import static com.segment.analytics.internal.Utils.assertNotNull;
+import static com.segment.analytics.internal.Utils.assertNotNullOrEmpty;
+import static com.segment.analytics.internal.Utils.isNullOrEmpty;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import com.segment.analytics.Properties;
+import com.segment.analytics.internal.Private;
+import java.util.Collections;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class AttemptGoalPayload extends BasePayload {
+
+    static final String EVENT_KEY = "Attempted Goal";
+    static final String PROPERTIES_KEY = "properties";
+
+    @Private
+    AttemptGoalPayload(
+            @NonNull String messageId,
+            @NonNull Date timestamp,
+            @NonNull Map<String, Object> context,
+            @NonNull Map<String, Object> integrations,
+            @Nullable String userId,
+            @NonNull String anonymousId,
+            @NonNull String event,
+            @NonNull Map<String, Object> properties) {
+        super(Type.attemptGoal, messageId, timestamp, context, integrations, userId, anonymousId);
```

Comment:
We want
 "type": "intervention"
to appear in the JSON

Reply:
done